adhocteam/Head-Start-TTADP
# Managers can add notes and set status of reports

**Description of change**
 * Review page is now different if the user looking at the page is the approving manager
 * Manager's review page includes the user's additional note and allows the manager to leave a manager note
 * Manager note added to DB
 * Report state is shown in the side nav's "review" section once it leaves "draft" mode
 * Policy, service and route added to update status of a report

**How to test**
1) Pull down, run migrations/seeders
2) Make sure the `CURRENT_USER_ID` is set to "1"
3) Fill out a complete report. Add "Hermione" as the approving manager
4) Refresh the report, since Hermione is user with the id 1 you should see the approving manager version of the review page.
5) Add a manager note and set the status. Note once you submit the status on the side nav `Review and submit` section updates
6) Refresh the page to verify the manager notes persists

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/138
* https://github.com/HHS/Head-Start-TTADP/issues/231
* https://github.com/HHS/Head-Start-TTADP/issues/236
* https://github.com/HHS/Head-Start-TTADP/issues/137

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
